internal/indexer/transactions.go

```diff
@@ -31,6 +33,13 @@ func (bc *blockContext) processTransactions(trxHashes *[]string) (err error) {
 					continue
 				}
 				internal := makeInternal(trx_model, entry)
+
+				err := UpdateBalancesInternal(&newAccounts, internal)
```

Comment:
You should also check genesis file and add balances that we have in the genesis

Reply:
Good point, added 👍🏼